neo4j/pom.xml

```diff
@@ -32,13 +32,6 @@
                 <artifactId>maven-processor-plugin</artifactId>
             </plugin>
             <plugin>
-                <artifactId>maven-enforcer-plugin</artifactId>
```

Comment:
I removed it because the enforcer is inherited form the parent pom. The reason it was included in this pom is because we decided to keep it disable. Now we don't have conflict anymore, so I want to be alerted when something changes